Assets/MixedRealityToolkit.Services/InputSystem/MixedRealityInputModule.cs

```diff
@@ -0,0 +1,316 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.EventSystems;
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+    [RequireComponent(typeof(Camera))]
+    public class MixedRealityInputModule : StandaloneInputModule, IMixedRealityPointerHandler, IMixedRealitySourceStateHandler
+    {
+#if UNITY_EDITOR
+        [UnityEditor.CustomEditor(typeof(MixedRealityInputModule))]
```

Comment:
Editor classes should be separated out into their own files and moved into the appropriate editor folders